app/main/controllers/privacy/RTMediaPrivacy.php

```diff
@@ -466,7 +466,7 @@ public function rt_privacy_settings_action() {
 		$default_privacy = sanitize_text_field( filter_input( INPUT_POST, 'rtmedia-default-privacy', FILTER_SANITIZE_STRING ) );
 		$nonce           = sanitize_text_field( filter_input( INPUT_POST, 'rtmedia_member_settings_privacy', FILTER_SANITIZE_STRING ) );
 
-		if ( ! empty( $default_privacy ) ) {
+		if ( ! empty( $default_privacy ) || '0' === $default_privacy ) {
```

Comment:
Could  we convert `$default_privacy` to `int` and then compare?

Reply:
It is stored as string and used at that everywhere. So i think it's fine here.

Reply:
Ok, I meant something similar to what is done here https://github.com/rtMediaWP/rtMedia/pull/1538/files#diff-3968235ba0fab09fdf96bb65f5a42a77R517

Reply:
oh okay. Changing it.